installer/pkg/config-generator/fixtures/kube-system.yaml

```diff
@@ -1,5 +1,49 @@
 apiVersion: v1
 data:
+  install-config: |
+    admin:
+      email: test@coreos.com
+      password: asd123
+    baseDomain: cluster.com
+    clusterID: ""
+    machines:
+    - name: master
+      platformConfig:
+        aws:
+          iamRoleName: ""
+          rootVolume:
+            iops: 100
+            size: 30
+            type: gp2
+          type: t2.medium
+      replicas: 3
+    - name: worker
+      platformConfig:
+        aws:
+          iamRoleName: ""
+          rootVolume:
+            iops: 100
+            size: 30
+            type: gp2
+          type: t2.medium
+      replicas: 3
+    metadata:
+      creationTimestamp: null
+      name: test
+    networking:
+      podCIDR:
+        IP: 10.2.0.0
+        Mask: //8AAA==
+      serviceCIDR:
+        IP: 10.3.0.0
+        Mask: //8AAA==
+      type: canal
+    platform:
+      aws:
+        region: eu-west-1
+        vpcCIDRBlock: 10.0.0.0/16
+        vpcID: ""
+    pullSecret: /path/config.json
```

Comment:
This is no longer useful.  The config should be updated to take the pull secret as a JSON string, even if users are allowed to provide it via a path.

Reply:
> just to clarify, the `types.InstallConfig.PullSecret` field is a string type that stores the contents of the pull secret file , which is a JSON string?

That's fine with me.  We expect to consume this as an opaque string, right?  So we should store it as a string (like the SSH pubkey) instead of parsing it out into a more detailed structure.